docs/reference/errors-and-warnings/NU3037.md

```diff
@@ -0,0 +1,23 @@
+---
+title: NuGet Warning NU3037
+description: NU3037 Warning code
+author: heng-liu
+ms.author: henli
+ms.date: 2/21/2019
+ms.topic: reference
+ms.reviewer: dtivel
+f1_keywords: 
+  - NU3037
+---
+
+# NuGet Warning NU3037
+
+
+### Issue
+
+The signature has expired as the certificate used to sign the package has expired.
+
+
+### Solution
+
+If in require mode, only packages signed by a trusted signer will pass validation. So please request the package author to re-sign the package using the `nuget sign` command as described in [NuGet docs](https://docs.microsoft.com/en-us/nuget/create-packages/sign-a-package) using a valid Timestamp Authority signing certificate.Otherwise, just ignore the warning.
```

Comment:
links to docs in the same docset should use relative paths: https://review.docs.microsoft.com/en-us/help/contribute/links-how-to?branch=master#links-to-articles-in-the-same-docset